pkg/controller/model/grafana_ingress.go

```diff
@@ -91,9 +93,10 @@ func GrafanaIngress(cr *v1alpha1.Grafana) *v1beta1.Ingress {
 	}
 }
 
-func ReconciledGrafanaIngress(cr *v1alpha1.Grafana, current *v1beta1.Ingress) *v1beta1.Ingress {
+func ReconciledGrafanaIngress(cr *v1alpha1.Grafana, current *ingressv1.Ingress) *ingressv1.Ingress {
 
 	reconciled := current.DeepCopy()
+	reconciled.APIVersion = current.APIVersion
```

Comment:
Does this have any significance in terms of the reconciliation ?

Reply:
I will remove it, i have added it for testing purpose, forgot to remove.

Reply:
@ujjwalchk-it , Addressed it.